modules/Microsoft.Network/virtualNetworkGateways/deploy.bicep

```diff
@@ -106,32 +106,28 @@ param tags object = {}
 @description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
 param enableDefaultTelemetry bool = true
 
-@description('Optional. The name of logs that will be streamed.')
+@description('Optional. The name of logs that will be streamed. "allLogs" includes all possible logs for the resource.')
```

Comment:
I guess this file is lacking the logs logic too?

Reply:
you're right. thanks. fixed